doc/api/util.md

```diff
@@ -424,6 +430,9 @@ changes:
     example below. **Default:** `true`.
 * Returns: {string} The representation of passed object
 
+Deprecation notice: The legacy function signature that was changed in v0.9.3
+`util.inspect(object, [showHidden], [depth], [colors])` has been deprecated.
```

Comment:
I’d put this closer to the end of the description… this is probably not relevant to a lot of people (and if it is, we shouldn’t deprecate it)